data/org.freedesktop.portal.Camera.xml

```diff
@@ -30,6 +30,9 @@
         @options: Vardict with optional further information
         @handle: Object path for the #org.freedesktop.portal.Request object representing this call
 
+        Request to gain access to the camera. The portal will present the user with a dialog to
+        allow or deny this request.
+
```

Comment:
I think only the "Request to gain access to the camera." is enough. The fact that a dialog may be presented is an implementation detail of the GNOME access portal.

Reply:
I added the dialog note as other portals note this. However those cases seem to be more substantial dialogs (file chooser, print etc). In this case it is a yes/no dialog and the other cases don't mention this, so agreed - will remove.